src/main/scala/com/microsoft/partnercatalyst/fortis/spark/analyzer/timeseries/ComputedAnomalyKmeansInputRecord.scala

```diff
@@ -0,0 +1,29 @@
+package com.microsoft.partnercatalyst.fortis.spark.analyzer.timeseries
+
+import com.microsoft.partnercatalyst.fortis.spark.dto.TopicCount
+import org.apache.spark.ml.linalg.{Vector, Vectors}
+
+case class ComputedAnomalyKmeansInputRecord(
+  period: String,
+  periodtype: String,
+  pipelinekey: String,
+  externalsourceid: String,
+  features: Vector
+) extends Serializable
+
+object ComputedAnomalyKmeansInputRecord {
+
+  def apply(topicCount: TopicCount,
+            parameters: ComputedAnomalyKmeansModelParameter): ComputedAnomalyKmeansInputRecord = ComputedAnomalyKmeansInputRecord(
+    topicCount.period,
+    topicCount.periodtype,
+    topicCount.pipelinekey,
+    topicCount.externalsourceid,
+    Vectors.dense(
+      parameters.periodFeature(topicCount),
```

Comment:
I don't think it's necessary - at least, in my testing, I saw Scala automatically doing the right thing, e.g.

```
scala> val a: Int = 30
a: Int = 30

scala> val b: Double = a
b: Double = 30.0
```